src/plugins/visualizations/public/visualize_app/components/viz_chart_warning.tsx

```diff
@@ -23,7 +23,7 @@ import type {
 
 interface Props {
   chartType: CHART_WITHOUT_SMALL_MULTIPLES | CHART_TO_BE_DEPRECATED;
-  chartConfigToken: string;
+  chartConfigToken?: string;
```

Comment:
Made this `chartConfigToken` optional because, for the deprecated input controls, there is currently no `uiSetting` toggle for it nor will there be a new visualization that replaces it - basically, the goal is that the input controls visualization will eventually be removed **entirely** rather than replaced with a newer version.